### PR TITLE
Update displayport_msp_bf_compat.c for more remove off ? @dji goggle

### PR DESCRIPTION
**Pull request by druckgott**
Fix some more Icons for DJI Integra to do not have a ?

### CODE REVIEW

File: src/main/io/displayport_msp_bf_compat.c [hunk 1]
```diff
@@ -65,32 +65,33 @@ uint8_t getBfCharacter(uint8_t ch, uint8_t page)
...
         return BF_SYM_HEADING;
+*/
 
     case SYM_SCALE:
-        return BF_SYM_SCALE;
+        return 'I';
 
     case SYM_HDP_L:
-        return BF_SYM_HDP_L;
+        return BF_SYM_RSSI;
```
Comment (MrD-RC) [inline src/main/io/displayport_msp_bf_compat.c:74]: I think using the RSSI symbol would be confusing. Maybe use 'H' for HDP_L and 'P' for HDP_R?

File: src/main/io/displayport_msp_bf_compat.c [hunk 2]
```diff
@@ -65,32 +65,33 @@ uint8_t getBfCharacter(uint8_t ch, uint8_t page)
...
 
     case SYM_HDP_R:
-        return BF_SYM_HDP_R;
-*/
+        return BF_SYM_RSSI;
```
Comment (MrD-RC) [inline src/main/io/displayport_msp_bf_compat.c:77]: See above comment for HDP_L

File: src/main/io/displayport_msp_bf_compat.c [hunk 3]
```diff
@@ -65,32 +65,33 @@ uint8_t getBfCharacter(uint8_t ch, uint8_t page)
...
     case SYM_HOME:
         return BF_SYM_HOMEFLAG;
 
     case SYM_2RSS:
         return BF_SYM_RSSI;
 
-/*
     case SYM_DB:
-        return BF_SYM_DB
+        return 'd' + 'B';
```
Comment (MrD-RC) [inline src/main/io/displayport_msp_bf_compat.c:86]: You can't do this. Symbols are a single space character. Adding multiple character will cause issues with the OSD creation.

File: src/main/io/displayport_msp_bf_compat.c [hunk 4]
```diff
@@ -65,32 +65,33 @@ uint8_t getBfCharacter(uint8_t ch, uint8_t page)
...
 
     case SYM_DBM:
-        return BF_SYM_DBM;
+        return 'd' + 'B' + 'm';
```
Comment (MrD-RC) [inline src/main/io/displayport_msp_bf_compat.c:89]: You can't do this. Symbols are a single space character. Adding multiple character will cause issues with the OSD creation.

File: src/main/io/displayport_msp_bf_compat.c [hunk 5]
```diff
@@ -65,32 +65,33 @@ uint8_t getBfCharacter(uint8_t ch, uint8_t page)
...
 
     case SYM_SNR:
-        return BF_SYM_SNR;
+        return 'S' + 'N' + 'R';
```
Comment (MrD-RC) [inline src/main/io/displayport_msp_bf_compat.c:92]: You can't do this. Symbols are a single space character. Adding multiple character will cause issues with the OSD creation.

File: src/main/io/displayport_msp_bf_compat.c [hunk 6]
```diff
@@ -192,13 +193,13 @@ uint8_t getBfCharacter(uint8_t ch, uint8_t page)
 */
     case SYM_WATT:
         return BF_SYM_WATT;
-/*
+
     case SYM_MW:
-        return BF_SYM_MW;
+        return 'M' + 'W';
```
Comment (MrD-RC) [inline src/main/io/displayport_msp_bf_compat.c:198]: You can't do this. Symbols are a single space character. Adding multiple character will cause issues with the OSD creation.

File: src/main/io/displayport_msp_bf_compat.c [hunk 7]
```diff
@@ -192,13 +193,13 @@ uint8_t getBfCharacter(uint8_t ch, uint8_t page)
...
 
     case SYM_KILOWATT:
-        return BF_SYM_KILOWATT;
-*/
+        return 'K' + 'W';
```
Comment (MrD-RC) [inline src/main/io/displayport_msp_bf_compat.c:201]: You can't do this. Symbols are a single space character. Adding multiple character will cause issues with the OSD creation.

**Review by MrD-RC — CHANGES_REQUESTED** [review]

**Comment by druckgott** [comment]
it dosend do sence currently with the points if you only have one letter.

File: src/main/io/displayport_msp_bf_compat.c (hunk 3 above)
Comment (rmaia3d) [reply src/main/io/displayport_msp_bf_compat.c:86]: Also, there's no support for lower case characters in the DJI font. Only upper-case.

**Pull request closed without merging**